content/500-support/100-help-articles/500-comparing-columns-through-raw-queries.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: 'Comparing columns of same table through raw queries'
+metaTitle: 'Comparing columns of same table through raw queries'
+metaDescription: 'Comparing columns with prisma's inbuilt raw query methods'
+---
+
+## Issue
+
+Comparing different columns from the same table is a common scenario that developers encounter. Some examples include comparing two numeric values in the same table or comparing two dates in a same table. There's an existing [GitHub Issue](https://github.com/prisma/prisma/issues/5048) regarding the same. 
+
+
+## Workaround
+
+Comparing values from two columns in the same table can be achieved by using [raw queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).
+
+### Comparing numeric values
+
+One use case for comparing values from different columns would be retrieving posts that have more comments than likes; in this case we need to compare the values of `commentsCount` and `likesCount`.
```

Comment:
```suggestion
One use case for comparing values from different columns would be retrieving posts that have more comments than likes; in this case, you need to compare the values of `commentsCount` and `likesCount`.
```

Reply:
The [docs style guide](https://www.prisma.io/docs/about/prisma-docs/style-guide#use-you-as-the-pronoun) suggests using the second person ("you") instead of first person ("we").